docs/data/base/pages.ts

```diff
@@ -24,7 +24,10 @@ const pages = [
       {
         pathname: '/base/components/utils',
         subheader: 'utils',
-        children: [{ pathname: '/base/react-click-away-listener', title: 'Click-away listener' }],
+        children: [
+          { pathname: '/base/react-click-away-listener', title: 'Click-away listener' },
+          { pathname: '/base/react-modal', title: 'Modal' },
+        ],
```

Comment:
I wonder if it wouldn't make more sense to have the Modal sit within the _Feedback_ sub-category?! I mean, you could think of everything in the Base package as utils, but the modal is ultimately about building feedback sort-of components.

Reply:
Interesting, in my opinion it is consider a util, because it is used for creating other components. You won't even likely as a developer use it directly in your application, you would use it for creating a dialog, alert or popovers etc. Does that makes sense?

Reply:
It does but still, you could think of any Base component as the foundation for more components, so in that sense, Base is essentially used for building other components. Therefore, everything could potentially be seen an util, I guess. `ClickAwayListener` is a very good example of a pure util but Modal is the foundational component of several others. 

I guess that having it under Feedback would also help somehow with discoverability.

Reply:
Alright, let's cc @michaldudak @siriwatknp for an opinion :)

Reply:
> It does but still, you could think of any Base component as the foundation for more components

The difference between Modal and, say, ButtonUnstyled is that you only need to provide styles for ButtonUnstyled and it's good to go. Modal is more of an internal building block.

> I guess that having it under Feedback would also help somehow with discoverability.

I don't think we want to "promote" Modal. We generally encourage devs to use Dialog as that's what they usually require (there's even a note in the docs: "If you are creating a modal dialog, you probably want to use the [Dialog](https://mui.com/components/dialogs/) component rather than directly using Modal")